From 350e59fa923bf90e64d447cb26e175e528459ef5 Mon Sep 17 00:00:00 2001
From: Sean Cross <xobs@kosagi.com>
Date: Thu, 14 Jan 2016 12:38:18 +0800
Subject: [PATCH 38/65] package-kernel: update to v4.4-novena-r3

The .config file had the IT6251 built as a module.  Make it a builtin.

Also, add some etnaviv cleanups.

Signed-off-by: Sean Cross <xobs@kosagi.com>
---
 package-kernel.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/package-kernel.sh b/package-kernel.sh
index 41b3fde..77e9355 100755
--- a/package-kernel.sh
+++ b/package-kernel.sh
@@ -1,7 +1,7 @@
 #!/bin/sh
 # A script to build the kernel .deb files, as used by Kosagi.
 threads=$(($(grep '^processor' /proc/cpuinfo  | cut -d: -f2 | tail -n1)+1))
-version=4.4-novena-r2
+version=4.4-novena-r3
 
 if git diff-index --name-only HEAD | grep -qv "^scripts/package"; then
 	echo "Repo has local changes.  Stash or commit them."
-- 
2.7.3

